collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,298 @@
+until the release.
+
+If you encounter any issue while testing my PR or while testing out the single-threaded samples builds, please report
+them [here](https://github.com/godotengine/godot/pull/85939).
+
+Finally, if you still intend to continue using threads on the Web platform, you can! And the next feature should aid you
+setup your game online.
+
+## Easy <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> <abbr title="Progressive Web App">PWA</abbr> for threaded builds
+With this new feature, if you still need to export your game with thread support, you'll are able to activate the necessary
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>/<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers
+without having to modify the server's header responses. You'll need to export your game as a Progressive Web App though.
+
+![Ensure Cross-Origin Isolation Headers export option](/storage/blog/progress-report-web-export-in-4-3/ensure-cross-origin-isolation-headers.webp)
+
+This is because this export option sets up the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
+build, to intercept the requests from the server and inject those missing
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
+
+## Boot splash screen
+Just before the beta feature freeze deadline, [@patwork](https://github.com/patwork), a new
+[#web](https://chat.godotengine.org/channel/web) contributor,
+included [some neat features](https://github.com/godotengine/godot/pull/91128):
+
+He cleaned up the game page code and he updated the progress bar to use the semantic (and very useful) HTML `<progress>`
+tag.
+
+But, mainly and from now on, he made the boot splash screen set in the project settings to be displayed on the screen
+while the engine and the assets are getting loaded by the browser.
+
+![New Godot Web loading screen](/storage/blog/progress-report-web-export-in-4-3/loading-screen.gif)
+
+If no boot splash screen has been set, the classic Godot boot splash will be seen, as it is currently displayed on
+other platforms.
+
+# Next steps
+Don't worry, these changes only mark the beginning for the Godot Engine to truly offer game developers a great and
+reliable way to create Web games.
+
+So work will be done to finalize and stabilize the samples playback for the Web. In the future, the node chain system
+should be powerful enough to support audio effects on buses, so we intend to work on this too.
+
+Based on the feedback we received at the <abbr title="Game Developers Conference">GDC</abbr> from partners and friends,
+we know that we need a way to reduce the size of our exports. Currently, the 4.3 release Web build .wasm is around 50MB.
```

Comment:
I have it at 40 MB here for the `web_nothreads_release` template, as of 4.3.dev4 (don't have newer templates installed to check quickly).

When compressed with Brotli, it's 5MB. Compressed with zip, it's 8MB.

So I wouldn't just say "50 MB" and let people assume that it means any time you download a Godot game you're downloading a 50 MB payload.

```suggestion
we know that we need a way to reduce the size of our exports. Currently, the 4.3 release Web build .wasm is around 40 MB uncompressed, and 5 MB compressed with Brotli.
```